channels/mx.m3u

```diff
@@ -80,8 +80,12 @@ https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhtjb-tdt.m3u8
 https://5ca3e84a76d30.streamlock.net/gpacifico3/smil:obregon.smil/chunklist_w1489788315_b978000_sleng.m3u8
 #EXTINF:-1 tvg-id="QuieroTV.mx" tvg-country="MX" tvg-language="" tvg-logo="" group-title="",Quiero TV (1080p) [Offline]
 https://quierotvlive.tk/httplive/playlist.m3u8
-#EXTINF:-1 tvg-id="RCGTV2.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/hzJu6QT.png" group-title="",RCG TV-2 (360p) [Not 24/7]
-http://wowzacontrol.com:1936/stream23/stream23/playlist.m3u8
```

Comment:
Working link
![image](https://user-images.githubusercontent.com/7253922/133588443-d3de21dd-f35c-4ff3-aca1-c8637492d923.png)

Reply:
ok not for me

i got also

RCG TV-3 (360p)
http://wowzacontrol.com:1936/stream56/stream56/playlist.m3u8